### ISSUE TITLE
Struct fields in pattern aren't formatted

### ISSUE THREAD

**Issue by cmbrandenburg** [rust-lang-nursery/rustfmt#827]
Rustfmt doesn't format struct fields as part of a pattern.

For example, Rustfmt ignores the following two `match` blocks—despite the whitespace differences around `{ .. }`.

```rust
match a {
    B::C { .. } => (),
}

match a {
    B::C{..} => (),
}
```

The first `match` block is the style used in The Book. Rustfmt should probably use that style.

**Comment by nrc** [comment]
Are you using a recent Rustfmt? This should have been fixed by #820.

**Comment by cmbrandenburg** [comment]
Not as recent as the fix! Sorry for the dupe
